inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -606,69 +605,67 @@ struct jit_uni_mvn_kernel_f32 : public jit_uni_mvn_kernel, public jit_generator
 //////////////////////////////////////////////////////////////////////////////////
 
 MKLDNNMVNNode::MKLDNNMVNNode(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache)
-        : MKLDNNNode(op, eng, cache), epsMode_(insideSqrt) {}
-
-void MKLDNNMVNNode::getSupportedDescriptors() {
-    if (!descs.empty())
-        return;
-
+        : MKLDNNNode(op, eng, cache) {
     std::string errPrefix = "MVN node with name '" + getName() + "' ";
 
-    auto cnnLayer = getCnnLayer();
-    if (cnnLayer == nullptr)
-        THROW_IE_EXCEPTION << errPrefix << "does not have CNN layer.";
-
-    if (getParentEdges().size() > 2)
-        THROW_IE_EXCEPTION << errPrefix << "has incorrect number of input edges.";
-
-    if (getChildEdges().empty())
+    if (op->get_output_size() == 0)
```

Comment:
As far as I understand `get_output_size` returns number of output ports, not edges. In this case we can compare with exact value taken from the specification: `1`.

Reply:
Fixed